qa/rpc-tests/zkey_import_export.py

```diff
@@ -38,8 +37,9 @@ def run_test(self):
         # the sender loses 'amount' plus fee; to_addr receives exactly 'amount'
         def z_send(from_node, from_addr, to_addr, amount):
             global fee
+            
             opid = from_node.z_sendmany(from_addr,
-                [{"address": to_addr, "amount": Decimal(amount)}], 1, fee)
+                [{"address": to_addr, "amount": Decimal(amount)}], 1, int(fee))
```

Comment:
Should this really be `int`?

`Decimal(str(fee))` seems more likely to be robust.

What if fee is changed from units of `zatoshis` to units of `ZEC`s?   Maybe the test _should_ break in that case, so maybe `int` is preferable.  I'm not sure.

Reply:
I think it's fine as it is. There's no need to use Decimal for zatoshis.